macros/src/config.rs

```diff
@@ -857,11 +858,12 @@ impl<'a> FieldConfig<'a> {
                 let constraints = self
                     .constraints
                     .const_expr(&self.container_config.crate_root)
-                    .unwrap_or_else(|| quote!(<_>::default()));
+                    .unwrap_or_else(|| quote!(#crate_root::types::Constraints::default()));
                 quote!(
+                    const #constraint_name : #crate_root::types::Constraints = #constraints;
                     encoder.encode_extension_addition(
                         #tag,
-                        <#ty as #crate_root::AsnType>::CONSTRAINTS.override_constraints(#constraints),
+                        <#ty as #crate_root::AsnType>::CONSTRAINTS.override_constraints(#constraint_name),
```

Comment:
I wonder if there's a benefit to making `override_constraints` const and removing that evaluation from runtime.

Reply:
It is quite hot function. I tried to make it const but there were some challenges related to `Cow` type. Another hot function is 

https://github.com/librasn/rasn/blob/4e10e6b46f8fa5f69adc2e0016a72caa700c0278/src/types/constraints.rs#L45-L47

which in theory could be also constant/pre-computed.